File: app/src/main/java/com/example/builday365/model/Timeline/TimeLineDao.java
package com.example.builday365.model.Timeline;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;
import androidx.room.Update;

@Dao
public interface TimeLineDao {

    @Query("SELECT * FROM TimeLine")

    TimeLine getAll();

    @Insert
    void insert(TimeLine timeLine);

    @Update
    void update(TimeLine timeLine);

    @Delete
    void delete(TimeLine timeLine);

}

File: app/src/main/java/com/example/builday365/model/timeline/TimeLineDao.kt
package com.example.builday365.model.timeline

import androidx.room.Dao
import androidx.room.Delete
import androidx.room.Insert
import androidx.room.Query
import androidx.room.Update

@Dao
interface TimeLineDao {
    @get:Query("SELECT * FROM TimeLine")
    val all: TimeLine?

    @Insert
    fun insert(timeLine: TimeLine?)

    @Update
    fun update(timeLine: TimeLine?)

    @Delete
    fun delete(timeLine: TimeLine?)
}